Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM python:3.9.9-buster AS node
+
+WORKDIR /opt/project
+
+EXPOSE 8555
+
+RUN set -xe \
+    && apt-get update \
+    && pip install \
+        pip==21.3.1 \
+        poetry==1.1.12
+
+COPY [ \
+    "LICENSE", \
+    "README.rst", \
+    "INSTALL.linux.rst", \
+    "INSTALL.macos.rst", \
+    "Makefile", \
+    "./" \
+]
+COPY node node
+
+COPY ["pyproject.toml", "poetry.lock", "./"]
+RUN make install
+
+COPY scripts/entrypoint.sh .
```

Comment:
be consistent. user either `.` or `./` everywhere

Reply:
It's possible to use only ./ when you copy several files in one COPY command. Let's use ./ everywhere.